src/App.js

```diff
@@ -1,15 +1,19 @@
 import './App.css';
 import { Route, Routes } from "react-router-dom";
 import Reviews from './pages/Reviews';
-import Categories from './pages/cateogries';
+import Categories from './pages/Cateogries';
+import Category from './pages/Category';
+
 
 function App() {
   return (
     <div className="App">
       {/* <header className="App-header"></header> */}
       <Routes>
+        {/* <Route path="/" element={} /> */}
       <Route path="/reviews" element={<Reviews/>} />
       <Route path="/categories" element={<Categories/>} />
+      <Route path="/category/:slug" element={<Category/>} />
```

Comment:
You should also be able to use the same component as the reviews one, as they're essentially the same thing. 

I presume, without visually seeing it. I'd guess they were the same, but one has filtered results.